### DISCUSSION TITLE
Welcome to AzureDeveloperTemplates Discussions!

### DISCUSSION THREAD

**Discussion by Daniel-Krzyczkowski** [Daniel-Krzyczkowski/AzureDeveloperTemplates#4] [General]
Do not hesitate to ask any questions you have about this project!

**Comment by RaviSourot** [comment]
Hey Daniel, 

I have am not so clear about some files created like in below -
1. AzureDeveloperTemplates.Starter.API\Core\Configuration
2. AzureDeveloperTemplates.Starter.API\Core\Middleware
3. AzureDeveloperTemplates.Starter.API\Core\AppBuilderExtensions

Could you plz explain me the use of these files. 

Thanks!
